.travis.yml

```diff
@@ -31,21 +31,17 @@ install:
   - export PATH="$HOME/miniconda/bin:$PATH"
   - hash -r
   - conda config --set always_yes yes --set changeps1 no
-  - conda update --yes --quiet conda
-  - conda install -n root conda conda-env anaconda-client notebook
+  - conda update conda
+  # So we can get python-coveralls
+  - conda config --append channels conda-forge
+  # The tests need to see the Python kernel in the root environment
+  - conda install conda-build anaconda-client ipykernel python-coveralls
+  # The tests also need to see an R kernel in another environment
+  - conda create -n test_env r-irkernel r-base
```

Comment:
Looks like you no longer specify `python=$TRAVIS_PYTHON_VERSION` anywhere, so all 3 test runs will be using the default python.

Reply:
Don't you want to `conda activate test_env` before the build?